templates/share-modal.php

```diff
@@ -1,41 +1,60 @@
-<?php 
-    $url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+<?php
+/**
+ * Share
+ *
+ * Share modal template
+ *
+ * @package WordPress
+ * @subpackage community-portal
+ * @version 1.0.0
+ * @author  Playground Inc.
+ */
```

Comment:
Code sniffer is asking for a single space after this line.

Reply:
Push changed

Reply:
Fixed